src/mem/vmem/vmem_alloc.c

```diff
@@ -114,6 +121,11 @@ mmu_pte_t *vmem_alloc_pte_table(void) {
 	log_debug("%p", addr);
 
 	return addr;
+#else
+	log_error("pte not using in this configuration");
```

Comment:
Replace this message with "PTE pool size is zero for current configuration", please.